zipkin-lens/src/components/Browser/TraceSummary.js

```diff
@@ -119,9 +119,10 @@ class TraceSummary extends React.Component {
     const { traceSummary } = this.props;
 
     const upperBarLabel = (
-      <div className="trace-summary__upper-bar-label">
-        <div>
-          {`${traceSummary.spanCount} spans ${traceSummary.durationStr}`}
+        <div className="trace-summary__upper-bar-label">
```

Comment:
Our strict linter output some errors :sob:
Please fix indentation of this line.